### DISCUSSION TITLE
xkbfile dependency

### DISCUSSION THREAD

**Discussion by grue-bleen** [eclipse-theia/theia#8580] [General]
on ubuntu 20.10:

Package xkbfile was not found in the pkg-config search path.
Perhaps you should add the directory containing `xkbfile.pc'
to the PKG_CONFIG_PATH environment variable
No package 'xkbfile' found
gyp: Call to '${PKG_CONFIG:-pkg-config} x11 xkbfile --cflags | sed s/-I//g' returned exit status 0 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1

installing libxkbfile-dev fixed it.

**Comment by vince-fugnitto** [comment]
@cyrilchemparathy are you suggesting updating our [prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites)?
If you don't mind, would you like to contribute the changes (it's currently referenced for `debian`)?

**Reply by grue-bleen** [reply]
certainly, happy to update prerequisites.  .travis.yml has the dependency, and a few more (libsecret-1-dev, xvfb, ...).  presumably should include all in prerequisites.
